File: examples/python/smoke_llm.py
"""Adapter smoke test: replays a fixture cassette. Run: python3 smoke_llm.py"""
from learnai.llm import complete

c = complete(
    "Reply with exactly: adapter smoke test ok",
    system="You are a test double. Reply with the exact text requested and nothing else.",
    max_tokens=32,
)
print(f"text={c.text!r} model={c.model} tokens={c.input_tokens}+{c.output_tokens} replayed={str(c.replayed).lower()} recorded={c.recorded_at}")

File: examples/ts/smoke_llm.ts
/** Adapter smoke test: replays a fixture cassette. Run: node smoke_llm.ts */
import { complete } from './src/learnai/llm.ts';

const c = await complete('Reply with exactly: adapter smoke test ok', {
  system: 'You are a test double. Reply with the exact text requested and nothing else.',
  maxTokens: 32,
});
console.log(`text='${c.text}' model=${c.model} tokens=${c.inputTokens}+${c.outputTokens} replayed=${c.replayed} recorded=${c.recordedAt}`);
